metadata-ingestion/src/datahub/telemetry/telemetry.py

```diff
@@ -158,6 +159,7 @@ def ping(
                 headers={
                     "user-agent": f"datahub {datahub_package.nice_version_name()}"
                 },
+                timouet=TIMEOUT,
```

Comment:
Shouldn't this be `timeout` rather than `timouet`?

Reply:
yes this was a typo. Thanks for noticing. Will fix